apps/web/modules/bookings/views/bookings-single-view.tsx

```diff
@@ -340,436 +394,558 @@ export default function Success(props: PageProps) {
-                      const field = eventType.bookingFields.find((field) => field.name === name);
-                      // We show location in the "where" section
-                      // We show Booker Name, Emails and guests in Who section
-                      // We show notes in additional notes section
-                      // We show rescheduleReason at the top
-                      if (!field) return null;
-                      const isSystemField = SystemField.safeParse(field.name);
-                      // SMS_REMINDER_NUMBER_FIELD is a system field but doesn't have a dedicated place in the UI. So, it would be shown through the following responses list
-                      // TITLE is also an identifier for booking question "What is this meeting about?"
-                      if (
-                        isSystemField.success &&
-                        field.name !== SMS_REMINDER_NUMBER_FIELD &&
-                        field.name !== TITLE_FIELD
-                      )
-                        return null;
-
-                      const label = field.label || t(field.defaultLabel || "");
-
-                      return (
-                        <>
-                          <div className="text-emphasis mt-4 font-medium">{label}</div>
-                          <p
-                            className="text-default break-words"
-                            data-testid="field-response"
-                            data-fob-field={field.name}>
-                            {field.type === "boolean" ? (response ? t("yes") : t("no")) : response.toString()}
-                          </p>
-                        </>
-                      );
-                    })}
-                  </div>
-                </div>
-                {requiresLoginToUpdate && (
-                  <>
-                    <hr className="border-subtle mb-8" />
-                    <div className="text-center">
-                      <span className="text-emphasis ltr:mr-2 rtl:ml-2">{t("need_to_make_a_change")}</span>
-                      {/* Login button but redirect to here */}
-                      <span className="text-default inline">
-                        <span className="underline" data-testid="reschedule-link">
-                          <Link
-                            href={`/auth/login?callbackUrl=${encodeURIComponent(
-                              `/booking/${bookingInfo?.uid}`
-                            )}`}
-                            legacyBehavior>
-                            {t("login")}
-                          </Link>
-                        </span>
-                      </span>
-                    </div>
                   </>
                 )}
-                {!requiresLoginToUpdate &&
-                  (!needsConfirmation || !userIsOwner) &&
-                  !isCancelled &&
-                  (!isCancellationMode ? (
+                {isFeedbackMode &&
+                  (noShow ? (
```

Comment:
Weird diffchecker fail from Github. This is the actual new part added, the rest got messed up in diffchecker probably because of added "!isFeedbackMode" check in other parts of the code